q2_sample_classifier/tests/test_classifier.py

```diff
@@ -1298,6 +1298,21 @@ def test_confusion_matrix(self):
         b = qiime2.CategoricalMetadataColumn(self.a)
         confusion_matrix(self.tmpd, self.a, b)
 
+    def test_confusion_matrix_dtype_coercion(self):
+        # NOTE: the targets are numbers but represented as str
+        test_targets = qiime2.CategoricalMetadataColumn(pd.Series(
```

Comment:
could you change the names of the test data to truth and predictions so that this is clearer?